packages/http-client-csharp/emitter/src/index.ts

```diff
@@ -1,4 +1,9 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
+export * from "./constants.js";
 export * from "./emitter.js";
+export * from "./lib/lib.js";
+export * from "./lib/log-level.js";
+export * from "./lib/logger.js";
+export * from "./options.js";
```

Comment:
Please ensure all the APIs that you are exporting are absolutely needed in the emitter in `autorest.csharp`.

I suggest we expose them by name instead `*` because it exposes everything we added later

Reply:
agree. Updated.